docs/installation/ksonnet.md

```diff
@@ -0,0 +1,67 @@
+# Installing Loki with Ksonnet
+
+## Prerequisites
+
+Make sure you have Ksonnet v0.8.0:
```

Comment:
Can you use newer version of ksonnet? I have installed ksonnet simply by using
```
brew install ksonnet/tap/ks
```
and got version 0.13.1.

Maybe installation instructions can be simplified?

If version 0.8.0 has to be used specifically, maybe we should mention that newer versions don't work, and why.

Reply:
That's a good question. @cyriltovena any idea if 0.8.0 is required?

Reply:
yes unfortunately. But we could instead may be advertised tanka ? which works with any tanka version ?

Reply:
+1 for Tanka 😛

Reply:
I'd like to replace it with Tanka but there's no `tk env add loki --namespace=loki` equivalent. I can have the docs say to copy `environments/default` and then modify spec.json but that doesn't seem as straightforward. 

WDYT @sh0rez?

Reply:
I'd rather add that command to Tanka. A handy way to create, list or clone environments would be a good fit.

Reply:
Open an issue, I will implement it

Reply:
Migrated the docs over to Tanka :tada: The only downside to using Tanka is that it doesn't default to the k8s API server of your current kubectl context, so there's a little more manual work involved than there used to be with `ks`.